infrastructure/simulation-tool/test/simulation.test.ts

```diff
@@ -0,0 +1,6 @@
+import { use } from 'chai';
+import sinonChai from 'sinon-chai';
+
+use(sinonChai);
+
+describe('simulation', () => {});
```

Comment:
this file does not exist in main anymore ( I reckon you should do the cherry pick after all)